velox/expression/FieldReference.cpp

```diff
@@ -20,6 +20,16 @@
 
 namespace facebook::velox::exec {
 
+void FieldReference::computeDistinctFields() {
+  SpecialForm::computeDistinctFields();
```

Comment:
Does FieldReference need to call the base's implementation?
I assume there would be no input_ or multiplyReferencedFields_.

Reply:
a FieldReference can have children if its accessing a field inside a struct, like, c0.col1, this will create 2 FieldReference expression (col1)<-(c0) where col1 is the parent of c0